changelog/v0.16.1/stop-prerelease-brew-prs.yaml

```diff
@@ -0,0 +1,4 @@
+changelog:
+  - type: FIX
+    description: Stop opening PRs to homebrew-core for non-stable versions, which is disallowed per https://docs.brew.sh/Acceptable-Formulae#stable-versions.
+    issueLink: https://github.com/solo-io/gloo/issues/3178
```

Comment:
`resolvesIssue: false` because the root issue still persists until this change is picked up in built bot